<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>examClass</class>
 <widget class="QMainWindow" name="examClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>exam</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 117, 175);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>361</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 6, 199);
font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>2</y>
      <width>991</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>8514oem</family>
           <pointsize>26</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 26pt &quot;8514oem&quot;;
background-color: rgb(170, 170, 255);
color: rgb(0, 117, 175);</string>
         </property>
         <property name="text">
          <string>Table of Stars</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTableView" name="tableView">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(155, 220, 255);</string>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>8514oem</family>
             <pointsize>20</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(0, 117, 175);
background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="text">
            <string>Search List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="list_search">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(155, 220, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
             </property>
             <property name="text">
              <string>RA</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
             </property>
             <property name="text">
              <string>DEC</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
             </property>
             <property name="text">
              <string>Diameter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="line_name">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(155, 220, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_ra">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(155, 220, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_dec">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(155, 220, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_diameter">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(155, 220, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
             </property>
             <property name="text">
              <string>Search: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_search">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(155, 220, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="check_const">
           <property name="styleSheet">
            <string notr="true">font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
           </property>
           <property name="text">
            <string>Only constellation</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="add_button_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 6, 199);
font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
         </property>
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 6, 199);
font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_button">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 6, 199);
font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
         </property>
         <property name="text">
          <string>SEARCH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="view_button">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 6, 199);
font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
         </property>
         <property name="text">
          <string>VIEW</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chart_button">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 6, 199);
font: 75 20pt &quot;8514oem&quot;;
color: rgb(232, 232, 232);</string>
         </property>
         <property name="text">
          <string>CHART</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="exam.qrc"/>
 </resources>
 <connections/>
</ui>
